### DISCUSSION TITLE
use of Express.js in Booster

### DISCUSSION THREAD

**Discussion by mmlngl** [boostercloud/booster#1529] [Q&A] [answered]
Just discovered Booster. Can't wait to try it out!

# regarding use of Express.js

Booster looks like it uses Express to create a local dev server – ok.
Everything on the web points to other platforms (Fastify, etc.) for performance reasons.

- Is Express deployed in production?
- Is that the right choice for Booster?
- what's the longterm attitude towards Express?

Thanks

**Comment by NickSeagull** [comment, marked as answer]
Hello! The usage of Express is only at the level of the local provider as the development environment as you mentioned. To deploy into other cloud providers, Express is not deployed, and instead what it is used is the native cloud capabilities. We're currently working on an alternative provider that should be able to be used as a local dev server and as well as an alternative to deploy dockerized apps. 

In that provider Express won't be used, and the current idea is to use Fastify indeed.

**Reply by mmlngl** [reply]
Clear!

Thanks for the clarification.
